analysis/model/src/main/kotlin/de/maibornwolff/codecharta/model/ProjectBuilder.kt

```diff
@@ -100,6 +100,7 @@ open class ProjectBuilder(
                 blacklist = blacklist.toList()
         )
 
+        System.err.println()
```

Comment:
Is this necessary?

Reply:
Well otherwise you have the log entry starting on the same line as the progress indicator - which looks odd

Reply:
... and unfortunately it does not work to let the log message (line 104) start with \n...